lib/cache/ObservableCollection.js

```diff
@@ -38,28 +36,6 @@ export default class ObservableCollection {
             cursorDescription.collectionName
         );
 
-        if (!this.collection) {
-            throw new Meteor.Error(
-                `We could not find the collection instance by name: "${
-                    this.collectionName
-                }", the cursor description was: ${JSON.stringify(
-                    cursorDescription
-                )}`
-            );
-        }
-
-        // Here we apply the logic of changing the cursor based on the collection-level configuration
-        if (this.collection._redisOplog) {
-            const { cursor } = this.collection._redisOplog;
-            if (cursor) {
-                const context = DDP._CurrentPublicationInvocation.get();
-                cursor.call(
-                    context,
-                    cursorDescription.options,
-                    cursorDescription.selector
-                );
-            }
-        }
```

Comment:
why remove this checks?

Reply:
They were moved to an `init` function, we just removed from the constructor.